src/widget/woverview.cpp

```diff
@@ -247,14 +249,36 @@ void WOverview::onEndOfTrackChange(double v) {
 
 void WOverview::onMarkChanged(double /*v*/) {
     //qDebug() << "WOverview::onMarkChanged()" << v;
-    update();
+    onTrackCueChange();
```

Comment:
Implicitly invoking update() in onTrackCueChange() is not intuitive. Please keep them separate and make the subtle difference between onMarkChange() and  onMarkChanged() obvious.

onTrackCueChange() should be renamed to updateCues() or updateCueMarks() to reflect what this function actually does an that it is not a slot.

Reply:
I don't even really know what that code does. It was there before, so I just left it so I don't accidentally break anything. Should I remove/refactor that part regardless?

Reply:
I don't quite get what you mean. There is no onMarkChange() function...